packages/ui-contract-editor/src/components/Clause/index.js

```diff
@@ -158,14 +158,15 @@ const ClauseComponent = React.forwardRef((props, ref) => {
           && header.length !== 0 ? <>
               <S.DragWrapper
                 {...iconWrapperProps}
+                style={{ marginBottom: '0px', position: 'relative', top: '-5px', maxHeight: '50px' }}
```

Comment:
Instead of adding these styles inline, I would add them to the styled component here: https://github.com/accordproject/web-components/blob/7ea0cb5e35c11fd45e786f3a56ea39854aa05ed0/packages/ui-contract-editor/src/components/styles.js#L146

Reply:
@DianaLease 
I have removed the inline styles and converted them to styled-components. Let me know if there are any more changes required. :)